doc/gallery/plot_rasterio_rgb.py

```diff
@@ -26,11 +26,9 @@
 # Read the data
 da = xr.open_rasterio('RGB.byte.tif')
 
-# Normalize the image
-da = da / 255
-
 # The data is in UTM projection. We have to set it manually until
-# https://github.com/SciTools/cartopy/issues/813 is implemented
+# https://github.com/SciTools/cartopy/issues/813 is implemented, so that
+# the coastlines are drawn in the same coordinate system as the image.
```

Comment:
This is not exactly what I meant. Right now we *can* draw the coastlines in the same coordinate system as the image. What we need https://github.com/SciTools/cartopy/issues/813 for is a tool to convert the PROJ4 string (available as attribute ``da.crs``) to a ``cartopy.Proj`` instance (see https://github.com/SciTools/cartopy/issues/813#issuecomment-355051930). To be honest I don't understand why Cartopy made the choice to create their own new projection classes with their own kwargs instead of relying on an existing system, but this is another topic...

Reply:
Right, I've just reverted this comment - I don't understand it well enough to improve the explanation, and it's not really part of this PR.